### ISSUE TITLE
I can't find the label ‘PATIENT’ and 'HER2FinalStatus' in the slidegraph.ipynb

### ISSUE THREAD

**Issue by Czzh1** [TissueImageAnalytics/tiatoolbox#747]
- TIA Toolbox version: 1.4.1
- Python version:3.8
- Operating System: Windows

### Description

![no_labels](https://github.com/TissueImageAnalytics/tiatoolbox/assets/83855684/d21d909a-c97c-459f-92e1-0de26dd702ed)
I want to run your code，but the dataset I downloaded from the TCGA-BRCA missed the label ‘PATIENT’ and 'HER2FinalStatus'

### What I Did
Anyone Can tell me what should I do?

**Comment by Czzh1** [comment]
How I can get the right dataset of TCGA-BRCA which is samed as yours?

**Comment by shaneahmed** [comment]
Which Jupyter notebook are you using?

**Comment by shaneahmed** [comment]
As this is training code, the notebook assumes you have the data for training.

Closing this issue. If you have any further questions, please feel free to reopen again.

**Comment by wangbo00129** [comment]
I'm trying to reproduce the results in tiatoolbox/examples/full-pipelines/slide-graph.ipynb and also need the exact CLINICAL_FILE mentioned in the notebook. Did you find it?

**Comment by Czzh1** [comment]
yes,I found it. So I hope you can provide the CLINICAL_FILE.I will thank you very much!&nbsp;



***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年2月27日(星期二) 下午2:12
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [TissueImageAnalytics/tiatoolbox] I can&#39;t find the label ‘PATIENT’ and &#39;HER2FinalStatus&#39; in the slidegraph.ipynb (Issue #747)





 
I'm trying to reproduce the results in tiatoolbox/examples/full-pipelines/slide-graph.ipynb and also need the exact CLINICAL_FILE mentioned in the notebook. Did you find it?

**Comment by wangbo00129** [comment]
I think I found the labels. There is a y in the graph jsons which is 1 or 0. I think these ys are the labels.
```
from glob import glob
import json
import os

def getLabelFromJson(path_json):
    print(path_json)
    label = json.load(open(path_json))
    return label

df_labels_from_json = pd.DataFrame()

for path_json in glob('graphs/*json'):
    name = os.path.basename(path_json).replace('.json', '')
    label = getLabelFromJson(path_json)['y']
    if isinstance(label, list) and len(label) == 1:
        label = label[0]
        df_labels_from_json.loc[name, 'label'] = label
    else:
        raise Exception(label)
        
```
